<launch>
        <node name="pioneer_atx_node" pkg="arduino_mr" type="pioneer_2at.py" />
        
        <node name="mtinode" pkg="xsens_driver" type="mtnode_new.py" output="screen">
             <param name="device" value="/dev/ttyUSB0"/>
             <param name="baudrate" value="115200"/>
        </node>
        
	<node pkg="joy" type="joy_node" name="joy" output="screen" />

	<node pkg="joy_to_arduino" type="teleop_MR_2at" name="joy_to_arduino" output="screen" />
	
</launch>
